src/Licenses/Entities/License/Predictions.php

```diff
@@ -0,0 +1,107 @@
+<?php
+
+namespace ArrowSphere\PublicApiClient\Licenses\Entities\License;
+
+use ArrowSphere\PublicApiClient\AbstractEntity;
+use ArrowSphere\PublicApiClient\Exception\EntityValidationException;
+
+/**
+ * Class Predictions
+ */
+class Predictions extends AbstractEntity
+{
+    public const COLUMN_CURRENCY = 'currency';
+    public const COLUMN_UPDATED_AT = 'updatedAt';
+    public const COLUMN_LICENSE_REFERENCE = 'licenseRef';
```

Comment:
I think you missed my comment here:
If I read the specs.yaml of your endpoint, this field is named "licenseReference"
```suggestion
    public const COLUMN_LICENSE_REFERENCE = 'licenseReference';
```